modules/internal/BytesStringCommon.chpl

```diff
@@ -546,7 +559,7 @@ module BytesStringCommon {
         }
       }
       // copy the data from rhs
-      bufferMemcpy(dst=lhs.buff, src_loc=rhs.locale_id, rhs.buff, rhsLen,
+      bufferMemcpy(dst=lhs.buff, src_loc=rhs.locale_id, rhs.buff, rhs.buffLen,
```

Comment:
removed `rhsLen` here since `rhs.buffLen` was used only once, can be used similarly like rhs.buff I think?